certs.jsonnet

```diff
@@ -15,8 +15,6 @@
   ],
 
   "eclipseprojects.io": [
-    # eclipseprojects.io does not have DNS
```

Comment:
with the recent changes, you the "CN" is not taken as a SAN, as sometime it's not even a valid DNS. so you need to add `eclipseprojects.io` in the array (e.g., see `polarsys.org ` or `theiacon.org `)